### ISSUE TITLE
Questions about LightNER paper

### ISSUE THREAD

**Issue by vasilisvyth** [zjunlp/DeepKE#92]
Hello, I have some questions about LightNER. 
When doing cross-domain few-shot learning did you used only one random sample for k-shot per entity when k is fixed?
Moreover,when further training your model for cross-domain few-shot learning, did you tune your hyperparameters,with the rest of the cross domain training data not included in k-shot, as validation set?
Thanks in advance.

**Comment by Flow3rDown** [comment]
Hi, thanks for your questions,
To alleviate the problem that a sentence may contain multiple entities, we control the total number of samples.
Last, we use the total validation set to tune our model to make the model more generalized in the DeepKE toolkit.
Thank you!

**Comment by cpmss521** [comment]
`we use the total validation set to tune our model` However, the original dataset MITMovie and MIT Restaurant don't have validation set.   How do you tune the model in this case? Thanks! @Flow3rDown

**Comment by Flow3rDown** [comment]
> `we use the total validation set to tune our model` However, the original dataset MITMovie and MIT Restaurant don't have validation set. How do you tune the model in this case? Thanks! @Flow3rDown

Hi, 
Since the Mit-Movie and MIT-Restaurant datasets have no validation set, we directly train the model without tuning parameters. A recommended way is to sample the validation set from the training set for parameter tuning.
Thanks for your attention!
